### PR TITLE
docs/op-guide: add two scaling files

### PR DESCRIPTION
**Pull request by lilin90**
Signed-off-by: lilin90 <lilin@pingcap.com>

## What have you changed? (mandatory)

Add two files about scaling the TiKV cluster.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

Relevant

## Does this PR affect tidb-ansible update? (mandatory)

No.

## Refer to a related PR or issue link (optional)

- https://github.com/pingcap/docs/blob/master/op-guide/ansible-deployment-scale.md
- https://github.com/pingcap/docs/blob/master/op-guide/horizontal-scale.md

### CODE REVIEW

**Comment by lilin90** [comment]
@LinuxGit PTAL

File: docs/op-guide/horizontal-scale.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: Scale a TiKV Cluster
+summary: Learn how to add or delete a PD/TiKV node.
+category: operations
+---
+
+# Scale a TiKV Cluster
+
+You can increase or decrease the capacity of a TiKV cluster without affecting online services. This document describes how to add or delete a PD or TiKV node.
```
Comment (rleungx) [inline docs/op-guide/horizontal-scale.md]: How about `scale in` and `scale out`?

Comment (lilin90) [reply docs/op-guide/horizontal-scale.md]: @rleungx You mean this sentence? "This document describes how to add or delete a PD or TiKV node."

File: docs/op-guide/ansible-deployment-scale.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+    | node5 | 172.16.10.5 | TiKV2 |
+    | node6 | 172.16.10.6 | TiKV3 |
+
+5. Perform a rolling update to the entire TiKV cluster:
+
+    ```
+    ansible-playbook rolling_update.yml
+    ```
+
+6. Update the Prometheus configuration and restart the cluster:
```
Comment (LinuxGit) [inline docs/op-guide/ansible-deployment-scale.md]: ditto

File: docs/op-guide/ansible-deployment-scale.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+title: Scale the TiKV Cluster Using TiDB-Ansible
+summary: Use TiDB-Ansible to add or delete a PD/TiKV node.
+category: operations
+---
+
+# Scale the TiKV Cluster Using TiDB-Ansible
+
+This document describes how to use TiDB-Ansible to increase or decrease the capacity of a TiKV cluster without affecting the online services. It applies to the TiKV deployment using Ansible. If your TiKV cluster is deployed in other ways, see [Scale a TiKV Cluster](horizontal-scale.md).
+
+> **Warning:** In decreasing the capacity, if your cluster has a mixed deployment of other services, do not perform the following procedures. The following examples assume that the removed nodes have no mixed deployment of other services.
```
Comment (Hoverbear) [inline docs/op-guide/ansible-deployment-scale.md]: Maybe this warning would be more effective in the section it's warning about.

File: docs/op-guide/horizontal-scale.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+### Add a TiKV node dynamically
+
+To add a new TiKV node dynamically, start a TiKV node on a new machine. The newly started TiKV node will automatically register in the existing PD of the cluster.
+
+To reduce the pressure of the existing TiKV nodes, PD loads balance automatically, which means PD gradually migrates some data to the new TiKV node.
+
+### Delete a TiKV node dynamically
+
+To delete (make it offline) a TiKV node safely, you need to inform PD in advance. After that, PD is able to migrate the data on this TiKV node to other TiKV nodes, ensuring that data have enough replicas.
```
Comment (Hoverbear) [inline docs/op-guide/horizontal-scale.md]: I think you can just say (offline) here and it's enough.

**Comment by lilin90** [comment]
@queenypingcap PTAL again. Thanks!

**Comment by queenypingcap** [comment]
LGTM. @LinuxGit PTAL again. Thanks!

**Comment by queenypingcap** [comment]
LGTM

**Pull request merged**